doc/guides/kconfig/index.rst

```diff
@@ -820,17 +820,21 @@ details on the functions.
 
 .. code-block:: none
 
-  dt_chosen_reg_addr(kconf, _, chosen, index=0, unit=None):
-  dt_chosen_reg_size(kconf, _, chosen, index=0, unit=None):
-  dt_node_reg_addr(kconf, _, path, index=0, unit=None):
-  dt_node_reg_size(kconf, _, path, index=0, unit=None):
+  dt_chosen_reg_addr_int(kconf, _, chosen, index=0, unit=None):
+  dt_chosen_reg_size_int(kconf, _, chosen, index=0, unit=None):
+  dt_chosen_reg_addr_hex(kconf, _, chosen, index=0, unit=None):
+  dt_chosen_reg_size_hex(kconf, _, chosen, index=0, unit=None):
+  dt_node_reg_addr_int(kconf, _, path, index=0, unit=None):
+  dt_node_reg_size_int(kconf, _, path, index=0, unit=None):
+  dt_node_reg_addr_hex(kconf, _, path, index=0, unit=None):
+  dt_node_reg_size_hex(kconf, _, path, index=0, unit=None):
```

Comment:
Maybe slightly easier to read in this order:

```
dt_chosen_reg_addr_int(kconf, _, chosen, index=0, unit=None):
dt_chosen_reg_addr_hex(kconf, _, chosen, index=0, unit=None):
dt_chosen_reg_size_int(kconf, _, chosen, index=0, unit=None):
dt_chosen_reg_size_hex(kconf, _, chosen, index=0, unit=None):
dt_node_reg_addr_int(kconf, _, path, index=0, unit=None):
dt_node_reg_addr_hex(kconf, _, path, index=0, unit=None):
dt_node_reg_size_int(kconf, _, path, index=0, unit=None):
dt_node_reg_size_hex(kconf, _, path, index=0, unit=None):
```

Reply:
done